### DISCUSSION TITLE
How to remove EC

### DISCUSSION THREAD

**Discussion by noinhotnn** [seaweedfs/seaweedfs#3634] [Q&A] [answered]
I removed ec in master.toml but data still exists and filer error "no free volumes left for" even though the hard drive is still empty (1T)
Master UI
![image](https://user-images.githubusercontent.com/13940206/189339613-1a91c8c1-6932-464a-9c90-6d0eeebb9029.png)
![image](https://user-images.githubusercontent.com/13940206/189339749-8819d7aa-15ce-4dd7-aa8c-d658fe0c488b.png)

**Comment by chrislusf** [comment, marked as answer]
see "help" within "weed shell" to find commands to convert EC volumes back to normal volumes.
